jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/WebSocketLocalEndpoint.java

```diff
@@ -28,25 +28,24 @@
  */
 public interface WebSocketLocalEndpoint
 {
-    Logger getLog();
+    Logger getLog(); // TODO why?
+    boolean isOpen(); // TODO Does the endpoint really know this?
```

Comment:
Yes, JSR356 has this requirement.

Reply:
I will endeavor to remove this at the websocket-core level.

Reply:
exactly, JSR356 stuff should be in the JSR356 API layer